README.md

```diff
@@ -121,3 +121,14 @@ is the following:
    but they can be run manually by clearing the `lookml_generator_staging` task in [Airflow](https://workflow.telemetry.mozilla.org/tree?dag_id=probe_scraper).
 3. Once the changes are merged and a new release has been pushed, re-run the DAG and the changes
    should take effect.
+
+## Generate Command Explained
+_high level explanation_
```

Comment:
IMO the "high level explanation" qualifier is unnecessary.
```suggestion
## `generate` Command Explained
```
Or maybe like this:
```suggestion
## `generate` command high level explanation
```